tests/lifecycle/tests/lifecycle_pod_recreation.go

```diff
@@ -0,0 +1,173 @@
+package tests
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/lifecycle/lifehelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/lifecycle/lifeparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/deployment"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/nodes"
+)
+
+var _ = Describe("lifecycle lifecycle-pod-recreation", func() {
+
+	BeforeEach(func() {
+		By("Clean namespace before each test")
+		err := namespaces.Clean(lifeparameters.LifecycleNamespace, globalhelper.APIClient)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Make masters schedulable")
+		err = lifehelper.EnableMasterScheduling(true)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Enable intrusive tests")
+		err = os.Setenv("TNF_NON_INTRUSIVE_ONLY", "false")
+		Expect(err).ToNot(HaveOccurred())
+
+	})
+
+	// 47405
+	It("One deployment with PodAntiAffinity, replicas < schedulable nodes", func() {
+		schedulableNodes := nodes.GetNumOfReadyNodesInCluster(globalhelper.APIClient)
```

Comment:
What if schedulableNodes==0 ?
Please add var validation by using Expect(len(schedulableNodes)).To(BeNumerically(">", 0), "Some error message!")